bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -253,7 +253,7 @@ channel-type.shelly.temperature4.description = Temperature of external Sensor #4
 channel-type.shelly.temperature5.label = Temperature 5
 channel-type.shelly.temperature5.description = Temperature of external Sensor #5
 channel-type.shelly.targetTemp.label = Target Temperature
-channel-type.shelly.targetTemp.description = Target Temperature in �C to be reached in auto-temperature mode
+channel-type.shelly.targetTemp.description = Target Temperature in °C to be reached in auto-temperature mode
```

Comment:
Something strange is happening here, I need to retry.

Reply:
This is shown in GitHub like that because the file was converted to UTF-8. It was in ANSI format before. To check in GitHub, click "View file", and it will now look correct.